### PR TITLE
refine travis-ci scripts

### PR DESCRIPTION
**Pull request by XenoAmess**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -306,14 +306,14 @@
...
           <quiet>true</quiet>
-          <encoding>${commons.encoding}</encoding>
+          <encoding>utf-8</encoding>
+          <docEncoding>utf-8</docEncoding>
           <notimestamp>true</notimestamp>
           <links>
-            <link>${commons.javadoc.java.link}</link>
-            <link>${commons.javadoc.javaee.link}</link>
+            <link>https://docs.oracle.com/javase/8/docs/api/</link>
+            <link>http://docs.oracle.com/javaee/6/api/</link>
```
Comment (kinow) [inline pom.xml:316]: I think these values must not be altered here. They are specified in the `commons-parent` module ([link](https://github.com/apache/commons-parent/blob/399b9eeb66d3b8cb030c9dd12099251b94a7f79a/pom.xml#L278))

**Review by kinow — CHANGES_REQUESTED** [review]
I think it would be better to modify only the CI scripts in this PR, leaving `pom.xml` for other PR (although I'm not sure if the changes are required, or OK here?)

**Comment by XenoAmess** [comment]
@kinow 
> I think it would be better to modify only the CI scripts in this PR, leaving `pom.xml` for other PR (although I'm not sure if the changes are required, or OK here?)

Got it. Sorry for that mistake.

**Review by kinow — APPROVED** [review]

**Comment by kinow** [comment]
Thanks @XenoAmess !

**Pull request merged**